app/models/swanson_quote_service.rb

```diff
@@ -0,0 +1,24 @@
+class SwansonQuoteService
+  include HTTParty
+  def initialize
+    @quote = get_quote
+  end
+  def get_quote
+      begin
+        #we get a random quote from ron-swanson_quotes
+        response = HTTParty.get('https://ron-swanson-quotes.herokuapp.com/v2/quotes')
+
+        return JSON.parse(response.body)[0]
+
+      rescue HTTParty::Error
+        return nil
+        # don´t do anything we handle this in the controller
```

Comment:
I personally would handle the errors/data manipulation here (for a larger project I may do another Service class for data manipulation) then just return success/fail and the data. Makes for easier testing and simple controllers or other methods.